### PR TITLE
fixed Responsiveness issue in login page

### PR DESCRIPTION
**Pull request by SayanDeveloper**
### Before:
oldPlone

### After:
newPlone

@fredvd @stevepiercy

### CODE REVIEW

File: news/3250.bugfix
Comment (stevepiercy) [inline news/3250.bugfix:1]: Give yourself credit!

```suggestion
Fixed responsiveness bug in login page. @SayanDeveloper
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
This PR requires review by a core developer before being merged. I requested review from a few, as it requires write permission to do so (and the Contributors group does not have that access).

**Comment by SayanDeveloper** [comment]
alright sir :-)

**Review by stevepiercy — APPROVED** [review]

File: theme/themes/pastanaga/extras/main.less
Comment (tiberiuichim) [inline theme/themes/pastanaga/extras/main.less:39]: I'm not comfortable with the `!important` here. Could you point to the setting that it's trying to "fix"? Why is the container working in the other pages?

Comment (SayanDeveloper) [reply theme/themes/pastanaga/extras/main.less:39]: because these stylings are getting inherited from semantic.less (which is not expected)
image

So i have done the override of styling for this specific login page container by specifying in ```#page-login > .container``` selector with !important

**Comment by davisagli** [comment]
Already fixed

**Pull request closed without merging**
